server/bin/www

```diff
@@ -23,21 +23,30 @@ var port = normalizePort(process.env.PORT || '8081');
 // var server = http.createServer(app);
 
 console.log("server was successfully created"); 
-var server; 
+
 /**
  * Listen on provided port, on all network interfaces.
  */
-function startListening() {
+function start() {
   var app = require('../app');
   app.set('port', port);
   console.log('app is running on port ', port); 
-  server = http.createServer(app);
+  var server = http.createServer(app);
   server.listen(port);
   server.on('error', onError);
-  server.on('listening', onListening);
 }; 
 
-connectToDB( startListening ); 
+var db = databaseModule.getDbInstance(); 
+
+db.on('error', function() {
+    console.log("Database connection error: " + DB_URI);
```

Comment:
I think we might need to do `console.log("Database connection error: " + databaseModule.DB_URI);` instead